exercises/04-integration/after/04-integration-after.sln

```diff
@@ -1,10 +1,7 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.26730.16
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Divergent.Customers", "Divergent.Customers\Divergent.Customers.csproj", "{02EE3114-734F-42DF-994C-98480E1489FE}"
-EndProject
```

Comment:
It looks like this file has become polluted due to the addition of `Customers.Host` in the first commit, and then the removal of the previous `Customers` and rename of `Customers.Host` to `Customers` in the second commit.

I think it's better if we treat this branch as a spike, to get feedback, but then the actual work is started again, on a fresh branch and in a new PR.

Reply:
I will reset the head and do a clean conversion 👍